rust/ql/lib/codeql/rust/elements/internal/MethodCallExprImpl.qll

```diff
@@ -43,10 +43,15 @@ module Impl {
     }
 
     override string toStringImpl() {
```

Comment:
It would have been nice to have all `toString` changes (including changes in `.expected` files) in a separate commit.